Spring-2019/stacks_queues/SOLUTIONS_2019Spring.md

```diff
@@ -36,21 +36,43 @@ If the problem is simple enough, remove this section. TODO :bug:
 
 TODO :bug:
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. PROBLEM 3 Validate Stack Sequences
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/validate-stack-sequences/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given two sequences pushed and popped with distinct values, return true if and only if this could have been the result of a sequence of push and pop operations on an initially empty stack.
```

Comment:
```suggestion
Given two sequences pushed and popped with distinct values, return true if and only if this could have been the result of a sequence of push and pop operations on an initially empty stack. Each item can only be pushed once and popped once.
```